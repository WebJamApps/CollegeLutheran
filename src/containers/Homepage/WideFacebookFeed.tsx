import React from 'react';

interface IWideFBFeed {
  width?: number;
}

const WideFacebookFeed = ({ width }: IWideFBFeed): JSX.Element => {
  let mb = '32px', mt = '0px';
  if (width && width < 1092) { mb = '0px'; mt = '32px'; }
  return (
    <div className="row" style={{ marginLeft: '10px' }}>
      <div className="col" style={{ maxWidth: '600px', margin: 'auto', paddingRight: '10px' }}>
        <p style={{ textAlign: 'center', fontSize: '10pt', marginTop: '0px' }}>
          <a style={{ fontSize: '10pt' }} href="/calendar">
            <i>View Full Calendar</i>
          </a>
        </p>
        <iframe // eslint-disable-next-line max-len
          src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&amp;showNav=0&amp;showDate=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;mode=AGENDA&amp;height=500&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&amp;color=%23060D5E&amp;ctz=America%2FNew_York"
          style={{ borderWidth: 0 }}
          width="550"
          height="500"
          frameBorder="0"
          scrolling="no"
          title="google-calendar"
        />
      </div>
      <div
        className="col"
        style={{
          width: '600px', margin: 'auto', paddingRight: '10px', paddingLeft: '10px', textAlign: 'center',
        }}
      >
        <p
          id="wideFacebook"
          style={{
            textAlign: 'center', fontSize: '10pt', marginBottom: mb, marginTop: mt,
          }}
        >
          <i>
            Like Us On
            {' '}
            <a style={{ fontSize: '10pt' }} href="https://www.facebook.com/CollegeLutheranChurch/">Facebook</a>
          </i>
        </p>
        <iframe
          className="widescreenHomepage"
          // eslint-disable-next-line max-len
          src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2FCollegeLutheranChurch&width=500&height=485&colorscheme=light&show_faces=false&header=true&stream=true&show_border=false"
          width="500"
          height="485"
          style={{ border: 'none', overflow: 'hidden', marginLeft: '-14px' }}
          scrolling="none"
          allow="encrypted-media"
          title="facebook ticker"
        />
      </div>
    </div>
  );
};
WideFacebookFeed.defaultProps = { width: 1004 };

export default WideFacebookFeed;
