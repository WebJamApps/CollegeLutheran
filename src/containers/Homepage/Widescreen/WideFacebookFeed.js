import React from 'react';

const WideFacebookFeed = () => (
  <div className="row" style={{ marginLeft: '10px' }}>
    <div className="col" style={{ maxWidth: '600px', margin: 'auto', paddingRight: '10px' }}>
      <p style={{ textAlign: 'center', fontSize: '10pt', marginTop: '0px' }}>
        <a style={{ fontSize: '10pt' }} href="/calendar">
          <i>View Full Calendar</i>
        </a>
      </p>
      <iframe // eslint-disable-next-line max-len
        src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&amp;showNav=0&amp;showDate=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;mode=AGENDA&amp;height=500&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&amp;color=%23060D5E&amp;ctz=America%2FNew_York"
        style={{ borderWidth: 0 }}
        width="550"
        height="500"
        frameBorder="0"
        scrolling="no"
        title="google-calendar"
      />
    </div>
    <div
      className="col"
      style={{
        width: '600px', margin: 'auto', paddingRight: '10px', paddingLeft: '10px', textAlign: 'center',
      }}
    >
      <p style={{
        textAlign: 'center', fontSize: '10pt', marginTop: '0px', marginBottom: '32px',
      }}
      >
        <i>
        Like Us On
          {' '}
          <a style={{ fontSize: '10pt' }} href="https://www.facebook.com/CollegeLutheranChurch/">Facebook</a>
        </i>
      </p>
      <iframe
        className="widescreenHomepage"
      // eslint-disable-next-line max-len
        src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2FCollegeLutheranChurch&width=500&height=485&colorscheme=light&show_faces=false&header=true&stream=true&show_border=false"
        width="500"
        height="485"
        style={{ border: 'none', overflow: 'hidden', marginLeft: '-14px' }}
        scrolling="none"
        allow="encrypted-media"
        title="facebook ticker"
      />
    </div>
  </div>
);

export default WideFacebookFeed;
