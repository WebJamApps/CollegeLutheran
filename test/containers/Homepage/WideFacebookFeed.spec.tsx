import WideFacebookFeed from 'src/containers/Homepage/WideFacebookFeed';
import renderer from 'react-test-renderer';

describe('WideFacebookFeed', () => {
  it('renders WideFacebookFeed', () => {
    const width = 90;
    const result = renderer.create(<WideFacebookFeed width={width} />).toJSON();
    expect(result).toMatchSnapshot();
  });
});
