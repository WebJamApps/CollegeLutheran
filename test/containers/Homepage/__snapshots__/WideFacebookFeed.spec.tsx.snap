// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WideFacebookFeed renders WideFacebookFeed 1`] = `
<div
  className="row"
  style={
    {
      "marginLeft": "10px",
    }
  }
>
  <div
    className="col"
    style={
      {
        "margin": "auto",
        "maxWidth": "600px",
        "paddingRight": "10px",
      }
    }
  >
    <p
      style={
        {
          "fontSize": "10pt",
          "marginTop": "0px",
          "textAlign": "center",
        }
      }
    >
      <a
        href="/calendar"
        style={
          {
            "fontSize": "10pt",
          }
        }
      >
        <i>
          View Full Calendar
        </i>
      </a>
    </p>
    <iframe
      frameBorder="0"
      height="500"
      scrolling="no"
      src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&showNav=0&showDate=0&showPrint=0&showTabs=0&showCalendars=0&mode=AGENDA&height=500&wkst=1&bgcolor=%23FFFFFF&src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&color=%23060D5E&ctz=America%2FNew_York"
      style={
        {
          "borderWidth": 0,
        }
      }
      title="google-calendar"
      width="550"
    />
  </div>
  <div
    className="col"
    style={
      {
        "margin": "auto",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "textAlign": "center",
        "width": "600px",
      }
    }
  >
    <p
      id="wideFacebook"
      style={
        {
          "fontSize": "10pt",
          "marginBottom": "0px",
          "marginTop": "32px",
          "textAlign": "center",
        }
      }
    >
      <i>
        Like Us On
         
        <a
          href="https://www.facebook.com/CollegeLutheranChurch/"
          style={
            {
              "fontSize": "10pt",
            }
          }
        >
          Facebook
        </a>
      </i>
    </p>
    <iframe
      allow="encrypted-media"
      className="widescreenHomepage"
      height="485"
      scrolling="none"
      src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2FCollegeLutheranChurch&width=500&height=485&colorscheme=light&show_faces=false&header=true&stream=true&show_border=false"
      style={
        {
          "border": "none",
          "marginLeft": "-14px",
          "overflow": "hidden",
        }
      }
      title="facebook ticker"
      width="500"
    />
  </div>
</div>
`;
